### ISSUE TITLE
Windows don't follow split-width-threshold

### ISSUE THREAD

**Issue by mgsb** [karthink/popper#74]
I use split-width-threshold to break a frame into multiple columns for windows. Opening a new popper window does "split" like other windows so it ends up spanning the whole frame. Is there a way to make new popper windows split like "normal" windows?

**Comment by karthink** [comment]
Yes, you can use a different `popper-display-function`, OR turn off Popper's window handling entirely (set `popper-display-control` to `nil` or `user`) and customize `display-buffer-alist`.

For the former option, you want a `display-buffer` action that does not use a side window.  (Side windows cannot be split.)  For example, you could try `display-buffer-below-selected` or `display-buffer-in-direction`.
